src/lib.rs

```diff
@@ -146,12 +147,12 @@ impl<S: Signer> CeramicHttpClient<S> {
     pub async fn create_single_instance_request(
         &self,
         model_id: &StreamId,
-    ) -> anyhow::Result<api::CreateRequest<DagCborEncoded>> {
+    ) -> anyhow::Result<api::CreateRequest<DeterministicInitEvent>> {
         if !model_id.is_model() {
             anyhow::bail!("StreamId was not a model");
         }
         let args = EventArgs::new_with_parent(&self.signer, model_id);
-        let commit = args.init()?;
+        let _commit = args.init()?;
```

Comment:
Was this wrong before? I don't really understand this change. Why do we need to do this if we don't use it? Is it to make sure it's valid or something?

Reply:
Yep, it's just a validity check. We don't actually use that information when making the deterministic init commit.